x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -89,7 +92,13 @@ export const SecuritySolutionTemplateWrapper: React.FC<SecuritySolutionPageWrapp
         $isTimelineBottomBarVisible={isTimelineBottomBarVisible}
         $isShowingTimelineOverlay={isShowingTimelineOverlay}
         bottomBarProps={SecuritySolutionBottomBarProps}
-        bottomBar={<SecuritySolutionBottomBar onAppLeave={onAppLeave} />}
+        bottomBar={
+          userHasSecuritySolutionVisible ? (
+            <SecuritySolutionBottomBar onAppLeave={onAppLeave} />
+          ) : (
```

Comment:
NIT: the `bottomBar` property is optional, we could also omit the prop or just pass `undefined`, instead of the empty fragment